### ISSUE TITLE
Some assets needed for scene-viewer example are missing

### ISSUE THREAD

**Issue by John-Nagle** [BVE-Reborn/rend3#310]
The scene-viewer example is missing some files in Github. Minor problem.

```
./scene-viewer
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: TextureIo("textures/Concrete-Color.png", FileError { path: "/home/john/projects/rend3/examples/scene-viewer/resources/default-scene/textures/Concrete-Color.png", error: Os { code: 2, kind: NotFound, message: "No such file or directory" } })', examples/scene-viewer/src/lib.rs:91:6

```
That's available from a previous version of Rend3. Fixed that.

Then

```
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: TextureIo("textures/BoomBoxWithAxes_baseColor.png", FileError { path: "/home/john/projects/rend3/examples/scene-viewer/resources/default-scene/textures/BoomBoxWithAxes_baseColor.png", error: Os { code: 2, kind: NotFound, message: "No such file or directory" } })', examples/scene-viewer/src/lib.rs:91:6

```
No sign of a boom box image in the rend3 files, current or older version.

**Comment by John-Nagle** [comment]
Tried to run an explict .gtlf file, per comments in scene-viewer:

./scene-viewer  ../../examples/scene-viewer/resources/default-scene/scene.gltf
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: TextureIo("textures/BoomBoxWithAxes_baseColor.png", FileError { path: "../../examples/scene-viewer/resources/default-scene/textures/BoomBoxWithAxes_baseColor.png", error: Os { code: 2, kind: NotFound, message: "No such file or directory" } })', examples/scene-viewer/src/lib.rs:91:6

**Comment by cwfitzgerald** [comment]
Ah whoops, the gltf data isn't even supposed to be there. When I changed test scenes, I thought I moved all of it into a separate download to prevent the repo size from growing too much, though apparently I missed the gltf files. There are instructions on downloading it in the scene viewer readme. I want to improve the error message here so that it's clear how to go about downloading the test scene.

I'm not sure if this is the best way of dealing with the test scene, but I know I don't want the data making the repo size giant
